tests/configs/config_ds001971.py

```diff
@@ -1,4 +1,4 @@
-"""Configuration file for the ds001971 dataset.
+"""Mobile brain body imaging (MoBI) gait adaptation experiment.
```

Comment:
Looking at the website I noticed that these descriptions were not great, so I made them at least a little bit better.